src/lxc/caps.c

```diff
@@ -247,6 +244,8 @@ bool lxc_file_cap_is_set(const char *path, cap_value_t cap, cap_flag_t flag)
 	cap_is_set = lxc_cap_is_set(caps, cap, flag);
 	cap_free(caps);
 	return cap_is_set;
+	#else
+	return true;
```

Comment:
Setting `errno = ENODATA` in `lxc_file_cap_is_set()` on `#ifnded LIBCAP_SUPPORTS_FILE_CAPABILTIES` is still worth it.